lib/api/src/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -356,23 +367,32 @@ instance IsServerError ErrMkTransaction where
         ErrMkTransactionTxBodyError hint ->
             apiError err500 CreatedInvalidTransaction hint
         ErrMkTransactionOutputTokenQuantityExceedsLimit e ->
-            apiError err403 OutputTokenQuantityExceedsLimit $ mconcat
-                [ "One of the token quantities you've specified is greater "
-                , "than the maximum quantity allowed in a single transaction "
-                , "output. "
-                , "Try splitting this quantity across two or more outputs. "
-                , "Destination address: "
-                , pretty (view #address e)
-                , ". Token policy identifier: "
-                , pretty (view (#asset . #policyId) e)
-                , ". Asset name: "
-                , pretty (view (#asset . #assetName) e)
-                , ". Token quantity specified: "
-                , pretty (view #quantity e)
-                , ". Maximum allowable token quantity: "
-                , pretty (view #quantityMaxBound e)
-                , "."
-                ]
+            flip (apiError err403) errorMessage $ OutputTokenQuantityExceedsLimit
+            ApiErrorOutputTokenQuantityExceedsLimit
+                { address = toText $ view #address e
+                , policyId = toText $ view (#asset . #policyId) e
+                , assetName = toText $ view (#asset . #assetName) e
+                , quantity = unTokenQuantity $ view #quantity e
+                , maxQuantity = unTokenQuantity $ view #quantityMaxBound e
+                }
+              where
+                errorMessage = T.unwords
+                    [ "One of the token quantities you've specified is greater "
+                    , "than the maximum quantity allowed in a single transaction "
+                    , "output. "
+                    , "Try splitting this quantity across two or more outputs. "
+                    , "Destination address: "
+                    , pretty (view #address e)
+                    , ". Token policy identifier: "
+                    , pretty (view (#asset . #policyId) e)
+                    , ". Asset name: "
+                    , pretty (view (#asset . #assetName) e)
+                    , ". Token quantity specified: "
+                    , pretty (view #quantity e)
+                    , ". Maximum allowable token quantity: "
+                    , pretty (view #quantityMaxBound e)
+                    , "."
```

Comment:
Perhaps we could just delete this section. What do you think?

Justification: all the relevant values are already included in the JSON record.

```suggestion
```